css/styles.css

```diff
@@ -54,3 +54,83 @@ button:active {
   background: white;
   border-width: 2px 2px 2px 2px;
 }
+
+.box1 {
+  background: #1a2236;
+  display: grid;
+  justify-items: center;
+  width: 320px;
```

Comment:
Kindly, your site should adapt to any size of mobile device, I recommend you not to limit your window with static measurements but use width 100%

Reply:
@mgmediaweb  Hi Gonzalo. I've done it the way you asked. But I thought this was not supposed to be responsive yet. After a certain screen size the design should change. If I change the size of the pictures with your approach it doesn't look good (I think). But I hope it complies with your requests.